client/pages/edit-profile-form.jsx

```diff
@@ -67,11 +67,11 @@ export default class EditProfileForm extends React.Component {
     formData.append('availability', this.state.availability);
     formData.append('image', this.fileInputRef.current.files[0]);
 
-    fetch('/api/users/3', {
+    fetch(`/api/users/${this.props.userId}`, {
       method: 'PUT',
       body: formData
     })
-      .then(response => response.json())
+      .then(res => res.json())
       .then(data => {
```

Comment:
Ok, for the PUT, we need to send your token as an `X-Access-Token` header so that users can't edit each other's profiles.